extensions/hibernate-orm/deployment/src/main/java/io/quarkus/hibernate/orm/deployment/HibernateOrmProcessor.java

```diff
@@ -934,11 +940,14 @@ public void test(CombinedIndexBuildItem index,
         }
     }
 
-    private Optional<String> guessDialect(Optional<String> dbKind) {
+    public static Optional<String> guessDialect(Optional<String> dbKind) {
```

Comment:
Since we don't have the same drivers for Hibernate Reactive, should it have its own code for this?

Reply:
We don't have the same drivers, but we do have the same dialects. The same SQL should be valid in either JDBC or Vertx. If that ever changes and we need to make special reactive-only dialects we can revisit this, but I don't foresee that happening.